### PR TITLE
add htmlcolors.rs

### PR DESCRIPTION
**Pull request by DancingGrumpyCat**
All W3 named colors in the same format as colors.rs.

### CODE REVIEW

File: htmlcolors.rs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	pub const DARKSLATEBLUE:    RGBColor = (72, 61, 139)
+	pub const MEDIUMSLATEBLUE:  RGBColor = (123, 104, 238)
+	pub const GREENYELLOW:      RGBColor = (173, 255, 47)
+	pub const CHARTREUSE:       RGBColor = (127, 255, 0)
+	pub const LAWNGREEN:        RGBColor = (124, 252, 0)
+	pub const LIME:             RGBColor = (0, 255, 0)
+	pub const LIMEGREEN:        RGBColor = (50, 205, 50)
+	pub const PALEGREEN:        RGBColor = (152, 251, 152)
+	pub const LIGHTGREEN:       RGBColor = (144, 238, 144)
+	pub const MEDIUMSPRINGGREEN RGBColor = (0, 250, 154)
```
Comment (trag1c) [inline htmlcolors.rs]: missed a colon here?

Comment (DancingGrumpyCat) [reply htmlcolors.rs]: oops ehehe

Comment (trag1c) [reply htmlcolors.rs]: missed it on L28 too

**Comment by qexat** [comment]
Hello, I appreciate your contribution!

Could you put these colors directly in `colors.rs` please? Also, I'd like a consistent code style across the project if possible :sweat_smile:
 
There is a missing newline before EOF as well, which should have normally been caught by pre-commit but I forgot to set it up

**Comment by DancingGrumpyCat** [comment]
is this better? ^^ should i not align the RGBColor = ... part?

**Comment by qexat** [comment]
> is this better? ^^ should i not align the RGBColor = ... part?

I'll fix the little issues myself don't worry

**Comment by DancingGrumpyCat** [comment]
> I'll fix the little issues myself don't worry

ok! 🙂

**Comment by DancingGrumpyCat** [comment]
there's some duplicate colors, i'll fix that in a sec

**Comment by qexat** [comment]
> there's some duplicate colors, i'll fix that in a sec

I'm fixing it already

**Comment by DancingGrumpyCat** [comment]
> I'm fixing it already

oh oops i didnt see that! you can see if my fix is good; i mostly removed the duplicates from the html colors version

**Comment by qexat** [comment]
> > I'm fixing it already
> 
> oh oops i didnt see that! you can see if my fix is good; i mostly removed the duplicates from the html colors version

Thank you for that ; though, I did it myself + cleaned up a bit and renamed few stuff so I removed your commit, sorry

**Comment by qexat** [comment]
Waiting for the CI to pass and I'll merge after

**Comment by DancingGrumpyCat** [comment]
sounds good!

**Pull request merged**